### PR TITLE
update `python3-deb` to 3.11

### PR DESCRIPTION
**Pull request by dorschw**
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10325

### CODE REVIEW

File: docker/gitlab-content-ci/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM demisto/python3-deb:3.10.11.61745
+FROM devdemisto/python3-deb:3.11.9.92379
```
Comment (JudahSchwartz) [inline docker/gitlab-content-ci/Dockerfile]: on purpose?

File: docker/python3/Pipfile.lock
Comment (JudahSchwartz) [inline docker/python3/Pipfile.lock:1]: probably shouldnt be in this pr

**Review by JudahSchwartz — APPROVED** [review]

File: docker/python3/Pipfile.lock
Comment (dorschw) [reply docker/python3/Pipfile.lock:1]: fixed

File: docker/python3-deb/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,6 @@
 
-# Last modified: 2024-07-04T00:14:12.140850+00:00
-
-FROM python:3.10.14-slim-bullseye
+# Last modified: 2024-06-14T00:13:48.572445+00:00
```
Comment (dorschw) [inline docker/python3-deb/Dockerfile:2]:
```suggestion
# Last modified: 2024-07-08T08:29:00+0000
```

**Pull request merged**